block_server.py

```diff
@@ -483,36 +363,33 @@ def _start_config_iocs(self):
         # Start the IOCs, if they are available and if they are flagged for autostart
         # Note: autostart means the IOC is started when the config is loaded,
         # restart means the IOC should automatically restart if it stops for some reason (e.g. it crashes)
-        for n, ioc in self._active_configserver.get_all_ioc_details().iteritems():
+        for name, ioc in self._active_configserver.get_all_ioc_details().iteritems():
             try:
                 # IOCs are restarted if and only if auto start is True. Note that auto restart instructs proc serv to
                 # restart an IOC if it terminates unexpectedly and does not apply here.
                 if ioc.autostart:
                     # Throws if IOC does not exist
-                    running = self._ioc_control.get_ioc_status(n)
-                    if running == "RUNNING":
-                        # Restart it
-                        self._ioc_control.restart_ioc(n)
-                    else:
-                        # Start it
-                        self._ioc_control.start_ioc(n)
+                    self._ioc_control.restart_ioc(name) \
```

Comment:
I don't like this style, blah if blah else should be for supper short things that fit on a line this took me a while to pass.